run-dev.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+docker-compose up -d
+docker-compose exec backend pip install -r dev-requirements.txt 
+docker-compose exec backend python manage.py migrate                     
+docker-compose exec backend python manage.py createorganisation 
+docker-compose exec backend python manage.py createsuperuser
```

Comment:
This means I have to create a new organisation and superuser every time I run this shell script. Probably not intended.

Reply:
As stated in the updated readme in case a user or org exists you can just Ctrl+C to ignore the commands, can instead create e.g. `init-dev.sh` which just runs the container without prompting for a creating of org and user.

Reply:
Fixed in most recent commit